src/nuclide.cpp

```diff
@@ -840,15 +835,14 @@ void Nuclide::calculate_urr_xs(int i_temp, Particle& p) const
   double r = future_prn(static_cast<int64_t>(index_), *p.current_seed());
   p.stream() = STREAM_TRACKING;
 
-  int i_low = 0;
-  while (urr.prob_(i_energy, URRTableParam::CUM_PROB, i_low) <= r) {
-    ++i_low;
-  };
-
-  int i_up = 0;
-  while (urr.prob_(i_energy + 1, URRTableParam::CUM_PROB, i_up) <= r) {
-    ++i_up;
-  };
+  // Warning: this assumes row-major order of cdf_values_. Also, should it
+  // really be +1?
```

Comment:
Let's get rid of the extra comment about +1
```suggestion
  // Warning: this assumes row-major order of cdf_values_
```